EIPS/eip-7378.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 7378
+title: Add time-weighted averaging to the base fee
+description: Using geometric weights to average past block sizes into consideration
+author: Guy Goren (@guy-goren) <guy.nahsholim@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/add-time-weighted-averaging-to-the-base-fee-mechanism/15142
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-07-22
+---
+
+## Abstract
+
```

Comment:
I think the abstract can be shorter something along the lines of (rest should be moved in specification)

`To prevent proposers/miners and/or users from colluding and gaming the base fee in the scenario of consecutive proposals this EIP proposes a new formula to update the base fee by considering geometric weighted past block sizes instead of just the last block size.`

Reply:
The motivation builds on the reader familiarity with the formula. I therefore, find it useful to have it right at the beginning. If you feel strongly on the matter, I can rearrange the presentation.